examples/demo_coordinate_system_datasets.ipynb

```diff
@@ -0,0 +1,916 @@
+    "\n",
+    "We also \n",
+    "- Compare the explicit calculation of a shear profile on the HSC Y3 source galaxies against a profile produced from `CLMM`. \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Instructions to download text data\n",
+    "\n",
+    "Before running this notebook, you will need to download some data.  The data is available through a [dropbox link](https://www.dropbox.com/scl/fo/dwsccslr5iwb7lnkf8jvx/AJkjgFeemUEHpHaZaHHqpAg?rlkey=efbtsr15mdrs3y6xsm7l48o0r&st=xb58ap0g&dl=0)\n",
+    "\n",
+    "First, create a directory where you want to put the example data, e.g. for a given `data_coords_dir`:\n",
+    "```\n",
+    "data_coords_dir=\"<YOUR PATH TO DATA COORDS DIR>/data_coords\"\n",
+    "mkdir -p $data_coords_dir\n",
+    "cd $data_coords_dir\n",
+    "```\n",
+    "Download all files from [dropbox link](https://www.dropbox.com/scl/fo/dwsccslr5iwb7lnkf8jvx/AJkjgFeemUEHpHaZaHHqpAg?rlkey=efbtsr15mdrs3y6xsm7l48o0r&st=xb58ap0g&dl=0).  This will be a zip file, `CLMM_data.zip` of size 242 Mb. scp or move this to `data_coords_dir`.\n",
+    "\n",
+    "From the directory, you should be able to unzip:\n",
+    "```\n",
+    "unzip data_CLMM.zip -d .\n",
+    "```\n",
+    "You now have the necessary data files to run this notebook.  \n",
+    "\n",
+    "**Make sure to change the `data_coords_dir` variable in the cell below to the appropriate location where you unzipped these files.**\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "#  CHANGE THIS TO YOUR LOCATION\n",
+    "data_coords_dir=\"/pscratch/sd/a/avestruz/clmm-area/CLMM_test_data/data_coords/\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "# %matplotlib inline\n",
+    "from astropy.table import Table\n",
+    "import pickle as pkl\n",
+    "from pathlib import Path\n",
+    "import pandas\n",
+    "from astropy.io import fits"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from clmm import Cosmology\n",
+    "cosmo = Cosmology(H0=70.0, Omega_dm0=0.27 - 0.045, Omega_b0=0.045, Omega_k0=0.0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example galaxy cluster from CosmoDC2\n",
+    "\n",
+    "Here, we plot an example galaxy cluster shear profile using `clmm`.  The cluster and source galaxy files are generated from the CosmoDC2 processed through TXPipe.  We test the coordinate system."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cluster = pandas.read_pickle(data_coords_dir+'test_cluster.pkl')"
```

Comment:
There's a '/' missing before 'test_cluster.pkl'

Reply:
`data_coords_dir` is defined with a "/" at the end.  We can add an additional one for user-foolproofing, but none of the read functions need this preceding the file name.

Reply:
Oh sorry, I must have deleted it on mine when changing my path ): then ignore all other comments